ui/src/app/settings/views/Repositories/RepositoriesList/RepositoriesList.js

```diff
@@ -100,6 +101,9 @@ export const Repositories = () => {
     repositorySelectors.search(state, searchText)
   );
   const dispatch = useDispatch();
+
+  useWindowTitle("Repositories");
```

Comment:
This is "Package repos" in the sidenav - should we keep them consistent?

Reply:
Thanks!